tests/inputData.json.example

```diff
@@ -34,6 +34,13 @@
 		"NSX Edge Nodes": [
 			"sfo-m01-en01",
 			"sfo-m01-en02"
+		],
+		"NSX Edge Fqdn": [
+			"sfo-m01-en01.sfo.rainpole.io",
+			"sfo-m01-en02.sfo.rainpole.io"
+		],
+		"Virtual Center": [
```

Comment:
Please replace instances of "Virtual Center" with "vCenter Server" in the JSON and the tests since that is the correct name.

Reply:
Taken care